Security/src/AuthClient/Controllers/HomeController.cs

```diff
@@ -0,0 +1,118 @@
+using Microsoft.AspNetCore.Authentication;
```

Comment:
The project names in the Discovery sample are "Web" (from ASP.NET Web Application) and "Service" (project type is ASP.NET API, but functionally considered a microservice).

The advantage of using "Web" makes it easy to add a console-based project next to it in the future.

Reply:
I see your point, and I don't disagree about that change being useful in that scenario, but it also seems like a change we could defer to a point in time when we wish to add such a sample.

Reply:
That's true. On the other hand, now would be the best opportunity because right now we're both familiar with its setup. When I tried using Discovery (service consumption) from a console project, I identified several bugs, which were easy to fix _at that time_.

I've created https://github.com/SteeltoeOSS/Samples/issues/334#issuecomment-2180273140 to track this. I'll leave it up to you to decide whether it should be in this PR, though I think we should address it eventually, and consider adding WorkerService/Desktop/Blazor projects where it makes sense.

Reply:
Add #335 for follow up soon